wolfssl/wolfcrypt/settings.h

```diff
@@ -175,6 +175,9 @@
 /* Uncomment next line if building for using Apache mynewt */
 /* #define WOLFSSL_APACHE_MYNEWT */
 
+/* Uncomment next line if building for using ESP-IDF */
+/* #define WOLFSSL_ESPIDF */
+
```

Comment:
We should have another define in here for `WOLFSSL_ESPWROOM32`, right?  If we don't list it here, there's a chance people will overlook it.

Reply:
I agree with you. I am going to add WOLFSSL_ESPWROOM32 define in the file and remove the define in user_settings.h.

Reply:
Thanks